content-repo/extra-docs/packs/Change_Management.md

```diff
@@ -0,0 +1,299 @@
+
+*Note: The suggested fields names in Jira are optional. The fields to map in XSOAR are mandatory.* 
+
+| **Field to create in Jira** | **Fields to map in XSOAR** |
+| --- | --- |
+| Change request summary | Details |
+| Change request description | Description |
+| Destination Port | Dst Ports |
+| FW policy action | Policy Actions |
+| FW Source Zone | Source Networks |
+| FW Destination Zone | Destination Networks |
+| IP Protocol | Protocol |
+| FW Service | Protocol names |
+| Request IP DST | Source IPs |
+| Request IP SRC | Destination IPs |
+
+To create a project in Jira, see https://support.atlassian.com/jira-software-cloud/docs/create-a-new-project/.
+To create custom fields in a Jira project, see https://support.atlassian.com/jira-cloud-administration/docs/create-a-custom-field/.
+
+
+
+
+2. In Jira, create an issue that will be used for mapping the custom fields in Cortex XSOAR. 
+3. Record the values of these fields for later use. 
+4. In Cortex XSOAR, fetch this specific incident. You can use the Query in the integration settings (in JQL).
+5. After the incident is fetched, go to **Setting** > **Object Setup** > **Classification and Mapping** and choose **classifier-mapper-incoming-JiraV2**.
+6. Click **Duplicate**.
+
+   ![Incidents](../../../docs/doc_imgs/reference/Change_Management/classification_and_mapping_Jira.PNG)
+
+
+
+7. Edit the duplicated mapper by clicking it.
+8. From the **Select Incident** dropdown list, select the incident you fetched.
+
+
+   ![Select_Instance](../../../docs/doc_imgs/reference/Change_Management/Jira_Mapper_load_json.PNG)
+
+
+
+
+9. Load the incident JSON file and search for the values that you previously recorded and map them accordingly. For example, if the value of *Request IP SRC* was 1.1.1.1 in Jira, search for it in the JSON file and map it to the relevant field. Repeat this for all the custom fields.
+
+   ![Map_fields](../../../docs/doc_imgs/reference/Change_Management/Jira_Map_fields.PNG)
+
+
+10. Create an instance of the **Atlassian Jira v2** integration.
+   1. Select **FW change management** as the incident type.
+   2. Select the mapper that was created above.
+
+     ![Instance_setting](../../../docs/doc_imgs/reference/Change_Management/Jira_settings.PNG)
+
+   3. For the outgoing mapper, select the default **classifier-mapper-outgoing-Jira**.
+   4. In the Query section, enter a query to ensure the only change management incidents are fetched. (In the example below, *cm* is the detected project for change management.)
+
+     ![Query](../../../docs/doc_imgs/reference/Change_Management/Jira_query.PNG)
+
+11. Since it is important to document all the processes in the playbook for future reference, if you are using the Jira integration, make sure all the relevant mirroring settings are enabled (https://xsoar.pan.dev/docs/reference/integrations/jira-v2#configure-incident-mirroring).
+
+
+
+
+### ServiceNow
```

Comment:
Maybe make sure to remind that this is for incidents and not the dedicated SNOW module